include/alpaka/queue/Traits.hpp

```diff
@@ -41,6 +41,8 @@ namespace alpaka
     //!   If the event has previously been queued, then this call will overwrite any existing state of the event.
     //!   Any subsequent calls which examine the status of event will only examine the completion of this most recent
     //!   call to enqueue.
+    //!   If a queue is waiting for an event the event state of the recently captured state at the time of the API call
+    //!   to wait() will be used to release the queue.
```

Comment:
I do not understand this statement. What is the "recently captured state"? Who captures it? The task?

Reply:
I change the formulation. The state of the event at the API call time is used to build dependencies (wait for an event) equally to CUDA.